client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -310,6 +313,7 @@ class ActivityLogItem extends Component {
 	renderSummary() {
 		const {
 			allowRestore,
```

Comment:
Good points, and I do think showing/disabling does need to be separate. I like that we talk about a `prop` that is directly relevant for this component (`disabled`)rather than some state that isn't directly relevant for this component (`inProgress`).

How about this:
* `allowRestore` becomes `showRestore`
* `disableRestore` is added